operator/pkg/ingress/ingress_reconcile.go

```diff
@@ -252,6 +252,11 @@ func (r *ingressReconciler) getSharedListenerPorts() (uint32, uint32, uint32) {
 }
 
 func (r *ingressReconciler) buildDedicatedResources(ctx context.Context, ingress *networkingv1.Ingress) (*ciliumv2.CiliumEnvoyConfig, *corev1.Service, *corev1.Endpoints, error) {
+	scopedLog := r.logger.WithFields(logrus.Fields{
```

Comment:
Until no we only logged in the `Reconcile` function - hence defining the scoped logger only there was enough. Not ideal that we have to redefine the scoped logger here again. I wonder whether it would be nicer to pass the scoped logger as additional method parameter (or as part of the context) :thinking: